libs/deps/mas/merch-card-collection.js

```diff
@@ -128,7 +128,7 @@ import{html as a,LitElement as A}from"/libs/deps/lit-all.min.js";var m=class{con
             <sp-action-menu
                 id="sortButton"
                 size="m"
-                @change="${this.sortChanged}"
+                onchange="${this.sortChanged}"
```

Comment:
I thought I already asked about this change? what is the reason?

Reply:
Yes, sorry, forgot to build mas. It should be good now